feature/foryou/src/main/kotlin/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreen.kt

```diff
@@ -160,6 +172,18 @@ internal fun ForYouScreen(
     )
     TrackScrollJank(scrollableState = state, stateName = "forYou:feed")
 
+    if(askToShowNotificationPermissionDialog){
+        ShowNotificationPermissionDialog(
+            onNotificationPermissionDismiss,
+            onNotificationPermissionConfirm
+        )
+    }
+
+    if(askNotificationPermission){
+        NotificationPermissionEffect()
+        onNotificationPermissionConfirm()
```

Comment:
This is a side-effect causing function called directly in composition - this should likely be wrapped in a `SideEffect` or something similar.